### PR TITLE
Return namespace from ScheduleAppsAndValidate method

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #https://portworx.atlassian.net/browse/PB-5883

**Special notes for your reviewer**:

### CODE REVIEW

File: apiServer/taas/utils/torpedoutils.go [hunk 1]
```diff
@@ -188,13 +188,19 @@ func ScheduleAppsAndValidate(c *gin.Context) {
...
+		namespacesList = append(namespacesList, namespace)
+	}
 	if len(errStrings) > 0 {
 		c.JSON(http.StatusInternalServerError, gin.H{
 			"error": errStrings,
 		})
 	} else {
 		c.JSON(http.StatusOK, gin.H{
-			"message": "Apps Created and Validated successfully",
+			"message":   "Apps Created and Validated successfully",
```
Comment (gvavinash) [inline apiServer/taas/utils/torpedoutils.go:202]: pls change to "App is created and validated successfully"

**Review by gvavinash — APPROVED** [review]
LGTM

File: apiServer/taas/utils/torpedoutils.go (hunk 1 above)
Comment (snigdha-px) [reply apiServer/taas/utils/torpedoutils.go]: done

File: apiServer/taas/utils/torpedoutils.go [hunk 2]
```diff
@@ -188,13 +188,19 @@ func ScheduleAppsAndValidate(c *gin.Context) {
...
+		namespacesList = append(namespacesList, namespace)
+	}
 	if len(errStrings) > 0 {
 		c.JSON(http.StatusInternalServerError, gin.H{
 			"error": errStrings,
 		})
 	} else {
 		c.JSON(http.StatusOK, gin.H{
-			"message": "Apps Created and Validated successfully",
+			"message":   "App is created and validated successfully",
```
Comment (mkoppal-px) [inline apiServer/taas/utils/torpedoutils.go:202]: nit: There is an extra space

**Review by mkoppal-px — APPROVED** [review]
LGTM except for a minor comment

File: apiServer/taas/utils/torpedoutils.go (hunk 2 above)
Comment (snigdha-px) [reply apiServer/taas/utils/torpedoutils.go]: removed

**Pull request merged**